Assignment2/bgDani-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+    "@context":{
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "pre": "http://www.oeg-upm.net/Organization#",
+        "vcard":"http://www.w3.org/2006/vcards/ns#",
+        "sen": "http://www.oeg-upm.net/Organization/Class01/includes/Sensor029#",
+        "comp": "http://www.oeg-upm.net/Organization/Class01/includes/Computer101#",
+        "meas": "http://www.oeg-upm.net/Organization/Class01/includes/Sensor029/Measurement8401#",
+        "us": "http://www.oeg-upm.net/Organization/Class01/includes/Computer101/User10A#",
+        "sensor": "rdfs:label",
+        "computer": "rdfs:label",
+        "measurement":"rdfs:label",
+        "user":"rdfs:label",
+        "date":"vcard:rev",
+        "num":"rdfs:label",
+        "name":"rdfs:label"
+    },
+    "@id":"http://www.oeg-upm.net/Organization/Class01",
+    "@type":"pre:Class01/includes",
```

Comment:
With "@type" you define the type of object or datatype. In this case, your object is Class01, thus you have to define it correctly.
In addition, "includes" represents the relationship (predicate) between your Class01 (suject) and the other objects (Sensor029 and Computer101), thus you should define it properly. Is something similar to the "geometry" property of the WiFi point example. You have to define it once.